dist/src/main/java/io/camunda/commons/clustering/DynamicClusterServices.java

```diff
@@ -37,31 +39,31 @@ public DynamicClusterServices(final ActorScheduler scheduler, final AtomixCluste
   }
 
   @Bean
-  GatewayClusterConfigurationService gatewayClusterTopologyService() {
+  @Profile("!broker")
```

Comment:
should we rather move this bean to `GatewayModuleConfiguration` ?

Reply:
Jaein. Basically, what I want to achieve is that I can start any node (with some exceptions) that includes the v2 endpoints. That means that the node is, to some degree, a gateway but without having the profile `gateway` set explicitly. Therefore, the node must join the cluster to retrieve the cluster topology.

If this would be moved to the `GatewayModuleConfiguration` which starts then the grpc endpoints, it would only initiate it only when the `gateway` profile is set. That's why I moved it out there to ensure that it gets initiated whenever the broker is not used.

Reply:
Maybe we rename it to ClusterTopologyService, to remove the confusion with the gateway